index.html

```diff
@@ -23,5 +23,7 @@ <h1>Dr. Norman Borlaug</h1>
     </Main>
 
     
+    
+    
```

Comment:
- I think you should avoid unnecessary empty lines, this would help improve readability.